scapy/main.py

```diff
@@ -270,6 +287,25 @@ def scapy_delete_temp_files():
         except:
             pass
 
+def _prepare_quote(quote, author, max_len=78):
```

Comment:
"""This function processes a quote and returns a string that is ready ti be used in the fancy prompt."""